packages/runtime/src/next/root-style-registry.tsx

```diff
@@ -7,8 +7,12 @@ import { ReactNode, createContext, useContext, useState } from 'react'
 
 const CacheContext = createContext(cache)
 
-const createRootStyleCache = () => {
-  const cache = createCache({ key: 'css' })
+const createRootStyleCache = (cacheKeyPrefix?: string) => {
```

Comment:
What do you think if we made this a named parameter and also had it define the entire cache key instead of just the prefix?